core/src/main/java/io/github/fourlastor/game/demo/round/step/BlobToss.kt

```diff
@@ -45,8 +60,22 @@ class BlobToss @AssistedInject constructor(
             null,
             null,
             null,
-            null, // Damage all mons on that tile.
-            Runnable { state.mons.forEach(Consumer { mon: Mon -> mon.changeHp(-2) }) }
+            null,
+            Runnable { state.mons.forEach(Consumer { mon: Mon -> mon.changeHp(-2) }) }, // Damage all mons on that tile.
```

Comment:
you should be able to change `forEach(Consumer { mon: Mon -> doStuffWith(mon) })` to `forEach { doStuffWith(it) }`